var/spack/repos/builtin/packages/precice/package.py

```diff
@@ -143,7 +143,7 @@ def variant_bool(feature, on='ON', off='OFF'):
             python_include = spec['python'].headers.directories[0]
             numpy_include = join_path(
                 spec['py-numpy'].prefix,
-                spec['python'].package.site_packages_dir,
+                spec['python'].package.platlib,
```

Comment:
Some packages are installed into purelib while others are installed into platlib. In the case of numpy, I'm guessing it's platlib since it isn't just pure Python code.

Most packages assume purelib at the moment, but that probably isn't true.

Reply:
I think we should make that change so packages have to specify platlib vs purelib, but I agree it can be put off for future work. I don't think it's necessary for the pip PR either, so not the same sort of rush.

Reply:
At the same time we should probably switch to prefix the python variables that we poke into module scope with `python_`